README.md

```diff
@@ -166,6 +174,42 @@ To create a `standard.x2` (or other non-default types) instance type, the `plan`
 managed_kafka.sh --create mykafka --plan standard.x2
 ```
 
+### Deploying to multiple clusters/zones
+In order to allow clients to deploy Kafka clusters to multiple clusters and/or zones, the following parameters must be customized:
+
+   * KUBE_CONFIG in `kas-fleet-manager-secrets-template-params`
+   * SUPPORTED_CLOUD_PROVIDERS in `kas-fleet-manager-service-template-params`
+   * CLUSTER_LIST in `kas-fleet-manager-service-template-params`
+
+The following sections describe the contents of these parameters in detail.
+
+#### SUPPORTED_CLOUD_PROVIDERS
+
+The SUPPORTED_CLOUD_PROVIDERS paramter contains a list of supported cloud providers in a yaml format.  See [deploying-kas-fleet-manager-to-openshift.md](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/docs/deploying-kas-fleet-manager-to-openshift.md) for more details.
+
+
+#### CLUSTER_LIST
+
+The CLUSTER_LIST parameter contains a list clusters for kas fleet manager.  See [data-plane-osd-cluster-options.md](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/docs/data-plane-osd-cluster-options.md) for more details.
+
+#### KUBE_CONFIG
+
+The KUBE_CONFIG contains cluster connection information for each cluster to which Kafka can be deployed.  There should be one entry in KUBE_CONFIG for each entry in CLUSTER_LIST.  For example, to generate a KUBE_CONFIG for two clusters:
```

Comment:
Might be worth putting a note that this is only needed when running in standalone mode. 
When using OCM, this is not needed. Same comment as https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/pull/187/files#r899930511